ras_party/models/models.py

```diff
@@ -360,3 +360,21 @@ class Enrolment(Base):
         ForeignKeyConstraint(['business_id', 'respondent_id'],
                              ['business_respondent.business_id', 'business_respondent.respondent_id']),
     )
+
+
+class PendingShares(Base):
+    __tablename__ = 'pending_shares'
+    email_address = Column(Text, primary_key=True)
+    business_id = Column(GUID, primary_key=True)
+    survey_id = Column(Text, primary_key=True)
+    time_shared = Column(DateTime, default=datetime.datetime.utcnow)
+    Index('pending_shares_business_idx', business_id)
+    Index('pending_shares_email_address_idx', email_address)
+    Index('pending_shares_survey_idx', survey_id)
+    Index('pending_shares_time_shared_idx', time_shared)
+
+    __table_args__ = (
+        ForeignKeyConstraint(['business_id'],
+                             ['business.party_uuid']),
+        UniqueConstraint('email_address', 'business_id', 'survey_id', name='u_constraint'),
```

Comment:
More of a question than something that needs to be changed, but is this needed?  Looking at the docs you can define a composite primary key by doing mutiple `primary_key=True` like you've done here. or is there something I'm not understanding.  It's probably the latter as I've not done much DB stuff lately!

Reply:
even I have not done it a lot so I am not sure. Although the sql script takes care of the unique constraint and I am not necessarily relying on sqlalchemy.

Reply:
Oki dokes, if it works then leave it as is.